doozer/doozerlib/cli/scan_osh.py

```diff
@@ -0,0 +1,114 @@
+import click
+import koji
+from doozerlib.cli import cli, click_coroutine, pass_runtime
+from doozerlib.runtime import Runtime
+from doozerlib.exectools import fire_and_forget
+from doozerlib.constants import BREWHUB_URL
+from doozerlib.util import cprint
+
+
+class ScanOshCli:
+    def __init__(self, runtime: Runtime, brew_tags: list, last_brew_event):
+        self.runtime = runtime
+        self.brew_tags = brew_tags
+        self.last_brew_event = last_brew_event
+
```

Comment:
dry-run support would be good here

Reply:
Done https://github.com/openshift-eng/art-tools/pull/51/commits/906d84751f15b00b02083b9cfe428b05c0888896